### PR TITLE
READ items in Database

### PR DESCRIPTION
**Pull request by vhanbrian**
PR title: [BE] Read Items From Database

Issue Link: 
https://app.asana.com/0/1202718774787420/1202734172666690

Definition of Done: 
	-Able to read items from the database with its corresponding room
Notes: 
	- All CRUD are all implemented already when creating the API 
	- Edit Read Items from API from id to value
	
Command: 
python manage.py runserver

Screenshots: 
BEFORE:
image

AFTER:
read_item

### CODE REVIEW

File: inventory_backend/api/serializers.py [hunk 1]
```diff
@@ -9,6 +9,13 @@ class Meta:
     fields = (['name'])
 
 class ReservationSerializer(serializers.ModelSerializer):
+  user= serializers.SlugRelatedField(
+     ## many = True,
+     ##read_only = False,
```
Comment (ziontavera) [inline inventory_backend/api/serializers.py]: Are these needed pa?

File: inventory_backend/api/serializers.py [hunk 2]
```diff
@@ -19,6 +26,14 @@ class Meta:
     fields = (['name'])
 
 class ItemSerializer(serializers.ModelSerializer):
+  room= serializers.SlugRelatedField(
+    ## many = True,
+    ##read_only = False,
```
Comment (ziontavera) [inline inventory_backend/api/serializers.py]: Are these needed pa?

**Review by ziontavera — COMMENTED** [review]
Please remove `pyc` and `env` files

File: inventory_backend/api/serializers.py (hunk 2 above)
Comment (vhanbrian) [reply inventory_backend/api/serializers.py]: I think its not needed anymore sir, will delete lang ba sir?

File: inventory_backend/api/serializers.py (hunk 1 above)
Comment (vhanbrian) [reply inventory_backend/api/serializers.py]: I think its not needed anymore sir, will delete lang ba sir?

**Review by ziontavera — COMMENTED** [review]
Please remove sqlite file. No need to include this when pushing

**Review by ziontavera — APPROVED** [review]

**Pull request merged**